lib/CMakeLists.txt

```diff
@@ -31,6 +31,10 @@ set(LinkedLibs
   TorchMLIRTorchOnnxToTorch
 )
 
+if(TORCH_MLIR_ENABLE_STABLEHLO)
+list(APPEND LinkedLibs StablehloPasses StablehloLinalgTransforms)
+endif()
```

Comment:
I'm having trouble compiling some part of this after syncing past this commit (f3e8199). My build is fine before this commit (8e2e5ee) and after this commit with `-DTORCH_MLIR_ENABLE_STABLEHLO=OFF`.

Error logs: https://gist.github.com/ScottTodd/332e4e76b744d0bfdd75c68cc8191970#file-torch_mlir_build-txt-L1784
System: Windows 10, MSVC (Visual Studio Build Tools 2022 Preview - amd64)

Do StableHLO and torch-mlir have Windows CI builds? I don't see issues in IREE's downstream Windows CI or locally, so either that is behind this version or something else is different about how the projects are configured.

Reply:
It seems a different behavior between different cxx compiler. If you do not need stablehlo backend, I think you could set `TORCH_MLIR_ENABLE_STABLEHLO=OFF` when building.

Reply:
Torch-MLIR doesn't have Windows CI builds now. I don't have an idea to fix this compiling error.

Reply:
It seems the same error as https://github.com/openxla/stablehlo/issues/1772